cabal-ghc901.project

```diff
@@ -45,9 +48,9 @@ source-repository-package
 
 source-repository-package
   type: git
-  location: https://github.com/jneira/hie-bios/
-  tag: 9b1445ab5efcabfad54043fc9b8e50e9d8c5bbf3
-  -- https://github.com/mpickering/hie-bios/pull/285
+  location: https://github.com/mpickering/hie-bios/
+  tag: 1875bff093983a0506f80e214eda27e7419da3bc
+  -- https://github.com/mpickering/hie-bios/pull/300
```

Comment:
is there an upstream issue to upload this to Hackage?

Reply:
I've commented in the mentioned pr about: https://github.com/mpickering/hie-bios/pull/300#issuecomment-906417219

Reply:
Release is done